brainframe/cli/translations/install.en.yml

```diff
@@ -22,8 +22,9 @@ en:
   data-path-help: "The path where BrainFrame will write data to at runtime"
   install-docker-help: "If provided, Docker will be automatically installed if
   it is not present. This is only available for supported operating systems."
-  install-curl-help: "If provided, curl will be automatically installed if not
-  present. This is only available for supported operating systems."
+  install-docker-compose-help: "If provided, Docker Compose will be
+  automatically installed if it is not present. This is only available for
+  supported operating systems."
```

Comment:
Do we want to explain what a "supported operating system" is?

Reply:
Turns out this isn't used anymore so I'll just remove it.